frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -182,8 +183,11 @@ export const VisualizationDirtyState = ({
     <RunButtonWithTooltip
       className="shadowed"
       circular
+      compact
+      style={{ width: 90 }}
       result={result}
       isRunnable={isRunnable}
+      isNative={isNative}
```

Comment:
What does the addition of `isNative` do here?

Reply:
I'd intended to pass this prop down to RunButtonWithTooltip to let it conditionally show a tooltip on the run button to let folks know what the shortcut key is for running a query, but I didn't finish that implementation, so I'll remove `isNative` in these spots.